pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/EventDao.sql.stg

```diff
@@ -51,7 +51,7 @@ LEFT JOIN message_destination md on ea.message_destination_id = md.message_desti
 LEFT JOIN activity_status_trigger ast ON ast.activity_status_trigger_id = ec.event_trigger_id
 LEFT JOIN activity_instance_status_type aist ON ast.activity_instance_status_type_id = aist.activity_instance_status_type_id
 LEFT JOIN workflow_state_trigger wst on wst.event_trigger_id = et.event_trigger_id
-LEFT JOIN dsm_notification_trigger dnt ON dnt.dsm_notification_trigger_id = et.event_trigger_type_id
+LEFT JOIN dsm_notification_trigger dnt ON dnt.dsm_notification_trigger_id = et.event_trigger_id
```

Comment:
This is a bug. Since the DSM notification triggers only matter for the `ReceiveDsmNotificationRoute`, I don't think this has caused any issues yet.